### ISSUE TITLE
Failed to register: [429] Rate limit exceeded: 429

### ISSUE THREAD

**Issue by jlo62** [AsamK/signal-cli#1225]
get this when i try to signup my number.
signal-cli -a +414455852xx register --voice:
`Failed to register: [429] Rate limit exceeded: 429`
I also get this error message on my laptop.

**Comment by AsamK** [comment]
you probably need to try to register with sms verification first, see #1202

**Comment by etlweather** [comment]
The SMS method of registration worked for me but once I got the 429, I had to try a different "clean" phone number. I don't know for how long the number that got the 429 was blocked for, I will have to give it a try again when I have some time.

**Comment by BootsSiR** [comment]
@etlweather did that number ever work for you?

**Comment by etlweather** [comment]
Where I use that, I have many landline numbers with the same area-code, and first 3 digits. So I tried one (before knowing about this) and got blocked. Then I tried another one (following the steps herein) and it worked. A couple weeks later, I tried the original number that got blocked with the new procedure and it worked.

So precisely "which number worked" - one of my landline numbers. These are real plain old landline from a phone service company.
